File: 프로그래머스/0/120912. 7의 개수/7의 개수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> array) {
    int answer = 0;
    
    for (int v : array) {
        string str = to_string(v);
        for (char c : str) {
            if (c == '7') answer++;
        }
    }
    return answer;
}

File: 프로그래머스/0/120912. 7의 개수/7의 개수.cs
using System;

public class Solution {
    public int solution(int[] array) {
        int answer = 0;
        string str;

        foreach (int v in array)
        {
            str = v.ToString();
            foreach (char c in str)
            {
                if (c == '7')
                    answer++;
            }
        }

        return answer;
    }
}